ignite/cmd/scaffold_band.go

```diff
@@ -1,94 +0,0 @@
-package ignitecmd
```

Comment:
If we want to keep the functionality until there is a plugin isn't it better just to hide the command using [Command.Hidden](https://pkg.go.dev/github.com/spf13/cobra#Command) instead of removing it? We just have to remember that once the plugin is implemented we have to actually remove the command.